app/views/articles/_video_player.html.erb

```diff
@@ -1,16 +1,28 @@
-<div itemscope itemtype="http://schema.org/VideoObject" class="video-player-header">
-  <% if meta_tags %>
-    <meta itemprop="uploadDate" content="<%= article.published_at %>" />
-    <meta itemprop="name" content="<%= article.title %>" />
-    <meta itemprop="description" content="<%= article.description %>" />
-    <meta itemprop="thumbnailUrl" content="<%= cloudinary(article.video_thumbnail_url, 880) %>" />
-    <meta itemprop="contentUrl" content="<%= article.video_source_url %>" />
-    <% minutes, seconds = article.video_duration_in_minutes.split(":") %>
-    <meta itemprop="duration" content="<%= format("PT%<minutes>sM%<seconds>sS", minutes: minutes, seconds: seconds) %>" />
-    <div id="video-player-source" data-source="<%= article.video_source_url %>"></div>
-  <% end %>
+<div class="video-player-header">
+  <%= article.published_at %>
+  <%= article.title %>
+  <%= article.description %>
+  <%= cloudinary(article.video_thumbnail_url, 880) %>
+  <%= article.video_source_url %>
```

Comment:
I suppose I should be removing this as well as line 8 since they were all `erb` in meta tags as well, correct @vaidehijoshi ?

Reply:
Confirm. It looks like they were previously being rendered dynamically within the `<meta>` tags, but since those are now removed, you're now just rendering them on the page. In fact, if you look at the vide player page locally, you'll see all this stuff rendered on the page and I'd guess that it breaks the way the view is rendered 😬So, you are right to remove them unless you intentionally wanted to render those on the view player page